lib/plugins/teams.js

```diff
@@ -14,20 +18,28 @@ module.exports = class Teams extends Diffable {
   }
 
   update (existing, attrs) {
-    return this.github.teams.addOrUpdateRepoInOrg(this.toParams(attrs))
+    return this.github.request(`PUT ${teamRepoEndpoint}`, this.toParams(existing, attrs))
   }
 
-  add (attrs) {
-    return this.github.teams.addOrUpdateRepoInOrg(this.toParams(attrs))
+  async add (attrs) {
+    const { data: existing } = await this.github.request(
+      'GET /orgs/:org/teams/:team_slug',
```

Comment:
dawned on me that this one isn't available either. i'll need to get this info in another way before this is ready

Reply:
actually, scratch that. looks like it is there, even in the now discontinued v2.17: https://developer.github.com/enterprise/2.17/v3/teams/#get-team-by-name